<launch>
    <arg name="mjcf" default="$(find mujoco_ros_sim)/franka_fr3/position_scene.xml"/>
    <arg name="camera_fps" default="60.0"/>
  
    <node pkg="mujoco_ros_sim"
          type="mujoco_sim_node.py"
          name="mujoco_sim_node"
          output="screen">
  
      <param name="robot_name" value="$(arg mjcf)"/>
      <param name="camera_fps" value="$(arg camera_fps)"/>
    </node>
  </launch>
  